src/systems/label/Label.hh

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#ifndef IGNITION_GAZEBO_SYSTEMS_LABEL_HH_
+#define IGNITION_GAZEBO_SYSTEMS_LABEL_HH_
+
+#include <memory>
+#include "ignition/gazebo/config.hh"
+#include "ignition/gazebo/System.hh"
+
+namespace ignition
+{
+namespace gazebo
+{
+// Inline bracket to help doxygen filtering.
+inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
+namespace systems
+{
+  /// \brief A label plugin that sets the label for the parent entity
+  class Label:
```

Comment:
Can you add some more documentation about how this plugin can be used in SDF? I'd like to have all of the SDF tag options specified (for this plugin, I believe the only tag is `<label>`), including valid/invalid input for each tag. Here's an example: https://github.com/ignitionrobotics/ign-gazebo/blob/d3ef3c2233a42ffeb771274b27eb3d6ace0dbf29/src/systems/logical_audio_sensor_plugin/LogicalAudioSensorPlugin.hh#L35-L131